README.md

```diff
@@ -1 +1,34 @@
-# kubevirt-ansible
\ No newline at end of file
+# kubevirt-ansible
+The purpose of this project to create Kubernetes or OpenShift cluster and deploy KubeVirt environment on it.
+Tested on CentOS Linux release 7.3.1611 (Core).
+
+### Requirements
+Install depending roles, and export `ANSIBLE_ROLES_PATH`
+```
+$ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
+$ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
+```
+For OpenShift deployment clone [**OpenShift Ansible project**](https://github.com/openshift/openshift-ansible)
+```
+$ git clone https://github.com/openshift/openshift-ansible
+```
+
```

Comment:
Please add an example of how I can deploy kubevirt to an existing cluster

Reply:
@fabiand Would you be OK if we continue and do polishing after we merge?

Reply:
Sure

Reply:
I've filed #5

Reply:
Thank you, lets do this :)